client/src/Game.cpp

```diff
@@ -118,15 +107,27 @@ sf::View Game::calculateViewport() {
 }
 
 void Game::handlePacket(const AresProtocol::AresMessage &message) {
-    switch (message.message_case()){
+    switch (message.message_case()) {
         case AresProtocol::AresMessage::kModifyObject:
             handleMsgModifyObject(message.modifyobject());
             break;
         default:
             break;
     }
 }
-
 void Game::handleMsgModifyObject(const AresProtocol::ModifyObject &modifyObject) {
-    //To be filled
+    if (modifyObject.action_case() == modifyObject.kCreate) {
```

Comment:
Thanks for the constant, it's a mistake.
I agree for the switch, this part was just a test.